x-pack/plugins/reporting/server/export_types/printable_pdf/lib/pdf/get_template.ts

```diff
@@ -29,6 +35,79 @@ export function getTemplate(
   const subheadingMarginBottom = 5;
   const subheadingHeight = subheadingFontSize * 1.5 + subheadingMarginTop + subheadingMarginBottom;
 
+  const getStyle = (): StyleDictionary => ({
```

Comment:
These next added functions were hoisted up from a large object literal down below that had to be simpler